### ISSUE TITLE
Add support for tabbing backwards on ListItem component

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/nextcloud-vue#2175]
I don't know if it fits here, but I just noticed that tabbing through the messages only works forwards. Tabbing backwards (with shift+tab) does not work.

_Originally posted by @raimund-schluessler in https://github.com/nextcloud/mail/issues/5281#issuecomment-901090147_
